src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-manager-message.service.ts

```diff
@@ -6,180 +6,136 @@ import { Task } from '../models/task';
 import { ServicesModule } from './services.module';
 
 class TaskManagerMessage {
-  descr: (metadata) => string;
-  running: (metadata) => string;
-  success: (metadata) => string;
-  error: (metadata) => object;
-
-  constructor(
-    descr: (metadata) => string,
-    running: (metadata) => string,
-    success: (metadata) => string,
-    error: (metadata) => object
-  ) {
-    this.descr = descr;
-    this.running = running;
-    this.success = success;
-    this.error = error;
+  errors: (metadata) => object;
+  prefixes: string[];
```

Comment:
Would it make sense if we had properties for each of the prefixes, instead of an array of strings?
It seams more clean to call `this.running` than `this.prefixes[1]`.
We could alternatively use an object/Model...